gateway/src/proxy.rs

```diff
@@ -72,7 +70,6 @@ where
 #[derive(Clone)]
 pub struct UserProxy {
     gateway: Arc<GatewayService>,
```

Comment:
We should be able to replace this with the Dal

Reply:
Done